src/code/z_actor.c

```diff
@@ -1,4 +1,5 @@
 #include "global.h"
+#include "quake.h"
```

Comment:
I think we should avoid including specific headers in source files for now, since we don't really gain anything by doing it now and it will be automated once we split global headers.
It's probably fine to keep this one since you already did the work, but at least I wouldn't bother doing it in the future